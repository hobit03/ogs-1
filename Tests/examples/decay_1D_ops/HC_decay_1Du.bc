GeoSys-BC: Boundary Conditions ------------------------------------------------
#BOUNDARY_CONDITION
 $PCS_TYPE
  GOUNDWATER_FLOW
 $PRIMARY_VARIABLE 
  PRESSURE1
 $GEO_TYPE
  POINT POINT1
 $DIS_TYPE
  CONSTANT: 0.0

#BOUNDARY_CONDITION
 $PCS_TYPE
  REACT_TRANS_OPS
 $PRIMARY_VARIABLE 
  Tracer
 $GEO_TYPE
  POINT POINT0
 $DIS_TYPE
  CONSTANT: 1.00

#BOUNDARY_CONDITION
 $PCS_TYPE
  REACT_TRANS_OPS
 $PRIMARY_VARIABLE 
  Decay_Father
 $GEO_TYPE
  POINT POINT0
 $DIS_TYPE
  CONSTANT: 1.00
#STOP
