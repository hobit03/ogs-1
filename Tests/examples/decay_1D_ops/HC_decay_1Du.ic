GeoSys-IC  Initial Conditions ------------------------------------------------
#INITIAL_CONDITION
 $PCS_TYPE
  GROUNDWATER_FLOW
 $PRIMARY_VARIABLE 
  HEAD
 $GEO_TYPE
  DOMAIN
 $DIS_TYPE
  CONSTANT 0.0

#INITIAL_CONDITION
 $PCS_TYPE
  REACT_TRANS_OPS
 $PRIMARY_VARIABLE 
  Tracer
 $GEO_TYPE
  DOMAIN
 $DIS_TYPE
  CONSTANT 0.0
  
#INITIAL_CONDITION
 $PCS_TYPE
  REACT_TRANS_OPS
 $PRIMARY_VARIABLE 
  Decay_Father
 $GEO_TYPE
  DOMAIN
 $DIS_TYPE
  CONSTANT 0.0
  
#INITIAL_CONDITION
 $PCS_TYPE
  REACT_TRANS_OPS
 $PRIMARY_VARIABLE 
  Decay_Daughter
 $GEO_TYPE
  DOMAIN
 $DIS_TYPE
  CONSTANT 0.0
#STOP

