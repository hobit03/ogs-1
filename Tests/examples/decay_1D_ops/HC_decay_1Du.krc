GeoSys Kinetic Reactions
#KINREACTIONDATA      
 $SOLVER_TYPE
	1                ; Solver type
 $RELATIVE_ERROR	
	1.e-6		 ; rel. Error
 $MIN_TIMESTEP	
	1.e-6		; min Timestep
 $INITIAL_TIMESTEP	
	1.e-4       ; init Timestep
 $BACTERIACAPACITY
  100.0
 $NO_REACTIONS
   POLYLINE PLY_BC_LEFT

#REACTION ; 
 $NAME
   first_order_decay ;
 $TYPE
   USER_EXP;
 $EQUATION
  1 Decay_Father = 1 Decay_Daughter ; 
 $USER_EXPRESSION
  2.0E-7 * mDecay_Father
 
#STOP