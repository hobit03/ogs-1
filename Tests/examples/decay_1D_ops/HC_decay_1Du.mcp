GeoSys-CP: Component Properties ------------------------------------------------

#COMPONENT_PROPERTIES  ; comp0
 $NAME
  Tracer ;
 $MOBILE 
  1; MOBIL-Flag: 0=immobile, 1=mobile/transported
 
#COMPONENT_PROPERTIES  ; comp0
 $NAME
  Decay_Father ;
 $MOBILE 
  1; MOBIL-Flag: 0=immobile, 1=mobile/transported

#COMPONENT_PROPERTIES  ; comp0
 $NAME
  Decay_Daughter ;
 $MOBILE 
  1; MOBIL-Flag: 0=immobile, 1=mobile/transported

#STOP
