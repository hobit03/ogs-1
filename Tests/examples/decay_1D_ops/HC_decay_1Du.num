GeoSys-NUM: Numerical Parameter ----------------------------------------
#NUMERICS
 $PCS_TYPE
  LIQUID_FLOW
 $ELE_GAUSS_POINTS
  3
 $LINEAR_SOLVER 
; method error_tolerance max_iterations theta precond storage
  2      6 1.e-014       1000           1.0   1       2
#NUMERICS
 $PCS_TYPE
  REACT_TRANS_OPS
 $LINEAR_SOLVER 
; method error_tolerance max_iterations theta precond storage
  2      6 1.e-014       1000           0.5   1       2
 $ELE_GAUSS_POINTS
  3
#STOP
