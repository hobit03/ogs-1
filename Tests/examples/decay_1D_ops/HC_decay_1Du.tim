GeoSys-TIM: Timestepping ------------------------------------------------
#TIME_STEPPING
 $PCS_TYPE
  GROUNDWATER_FLOW
 $TIME_STEPS
   100  86400.0
 $TIME_END
  8640000.0
 $TIME_START
  0.0
#TIME_STEPPING
 $PCS_TYPE
  REACT_TRANS_OPS
 $TIME_STEPS
   100  86400.0
 $TIME_END
  8640000.0
 $TIME_START
  0.0
#STOP

