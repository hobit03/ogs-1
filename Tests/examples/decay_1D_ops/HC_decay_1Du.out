GeoSys-OUT: Output ------------------------------------------------
#OUTPUT // domain nodes
 $NOD_VALUES
  HEAD
  Tracer
  Decay_Father
  Decay_Daughter
 $GEO_TYPE
  DOMAIN
 $DAT_TYPE
  PVD
 $TIM_TYPE
  STEPS 1
#STOP



 $TIM_TYPE
  0
  86400
  2160000
  4320000
  6480000
  8640000


